src/pages/Proposal/components/WithoutApprovalModal/index.tsx

```diff
@@ -0,0 +1,185 @@
+// eslint-disable-next-line no-use-before-define
+import React from "react";
+import {
+  CheckCircleOutlined,
+  CloseCircleOutlined,
+  CloseCircleFilled,
+  CheckCircleFilled,
+} from "@ant-design/icons";
+import { Modal } from "antd";
+import CopylistItem from "../CopylistItem";
+import "./index.less";
+import useMobile from "../../../../hooks/useMobile";
+
+interface IStatus {
+  // 0: success 1: fail 2: loading 3: un-arrival
+  verification: number;
+  execution: number;
+}
+interface IModalProps {
+  isUpdate: boolean;
+  transactionId: string;
+  message: string;
+  status: IStatus;
+  cancel: Function;
+}
+interface IProps {
+  open: boolean;
+  withoutApprovalProps: IModalProps;
+}
+
+const noticeDeployContent = [
+  "If the transaction pre-validation fails, fees will not be charged.",
+  "If the deployment fails, fees charged will not be returned.",
+  "Contract deployment includes 2 phases and takes around 1-10 minutes.",
+  "Closing deployment window while it's ongoing will not affect its progress.",
+];
+const noticeUpdateContent = [
+  "If the update fails, fees charged will not be returned.",
+  "Contract update includes 2 phases and takes around 1-10 minutes.",
+  "Contract deployment includes 2 phases and takes around 1-10 minutes.",
+];
+const getMessage = (props) => {
```

Comment:
Code Complete：Chapter 19
For each if to add 1 score. Find a way to simplify this.